libr/core/cmd_eval.c

```diff
@@ -356,6 +356,8 @@ static int cmd_eval(void *data, const char *input) {
 			v = strchr (k, '=');
 			if (v) {
 				*v++ = 0;
+				r_str_trim_head_tail (k);
```

Comment:
one of the issues i see in those trim apis is that sometimes they realloc the heap. and it end up in random uaf situations, this is not the case for this function because this one calls head and tail, which one memmoves and the other appends a null byte.

in this case this function should return void. to avoid this confusion. what do you think?

Reply:
Yeah it was indeed a bit confusing at first. I can change those API if you want, but it has to wait for Monday